src/settings/OpenExceptionalForm/ExceptionalPeriodEditor.js

```diff
@@ -138,8 +139,8 @@ class ExceptionalPeriodEditor extends React.Component {
 
     const items = this.state.servicePoints;
     const itemFormatter = (item) => (
-      <li data-test-service-point key={item.id}>
-        <div className="CircleDiv" style={{ background: item.color }} />
+      <li data-test-service-point key={item.id} class="service-point-selector-row">
```

Comment:
Change please attribute "class" to "className"
Also, to be honest, our team prefere use camelCase for class naming. So as we now responsible for this module, could you please use this style for all clasess that you change?
Thanks!

Reply:
Sure!  I saw just about every casing style throughout the repo, so assumed dash-case incorrectly, and will fix this.